sql-aggregates/profit-margin.sql

```diff
@@ -0,0 +1,4 @@
+select count("filmId") as "totalCopiesOfFilm"
+  from "inventory"
+  group by "filmId"
+  order by "filmId";
```

Comment:
Your optional challenge solution seems to be missing profit information:

---

![image](https://user-images.githubusercontent.com/16143771/180912105-c7d60cbe-6d7a-4061-86a6-851a5d0c9d23.png)